Sming/Libraries/MPU6050/component.mk

```diff
@@ -0,0 +1 @@
+COMPONENT_DEPENDS := I2Cdev
```

Comment:
To make doxygen comments available (from .h files) add `COMPONENT_DOXYGEN_INPUT := .`